storybookjs/storybook
# Addon-docs: Support absolute anchors when deployed at non-root

Issue: #11404

## What I did

When Storybook is deployed at an URL like `https://example.com/storybook/component-library/v48.3.2/`, the anchors within docs content will remove the base url.

E.g. when you're on `https://example.com/storybook/component-library/v48.3.2/?path=/docs/components-icon--default` and you have a link to `/docs/components-button--default` in the documentation, the resulting link will point to `https://example.com/?path=/docs/components-button--default`.

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

Not sure if there's a good way to add a test for this, as it depends on a special kind of deployment?
